.github/workflows/update_repos.yaml

```diff
@@ -0,0 +1,23 @@
+name: Update Repos
+on:
+  workflow_dispatch:
+  push:
+    branches: ["main"]
+jobs:
+  update-repos:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup deno
+        uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x
+      - name: Run deno grab action
+        run: deno task grab
+      - uses: stefanzweifel/git-auto-commit-action@v4

```

Comment:
Sorry, but running 3rd party GitHub Actions tasks is basically a non-starter... for the same reasons we couldn't approve of the binary images. We can't really trust 3rd party code running on our repos, especially with permission to alter their contents, as this would necessarily need to do.

The code that would need to run would have to be included in this repo, or use standard GitHub.com created actions (or equally respected and widely used, like the ruby/ actions).

Reply:
What do you consider as "equally respected and widely used?" Would Node.js work instead of Deno? If so, I can easily modify the code to use Node.js instead.

Reply:
Ideally, all the code that runs on the project and alters the project is contained within this repo. The GitHub Actions could run a python script checked into this repo, for example. I would prefer basic shell scripts (bash) or python, because we need to keep it to something that most developers could maintain, and not do anything too niche. Lowering the barrier to maintenance is the best way to ensure that we'll be able to keep this maintained, when people switch jobs, move positions, prioritize other projects, etc.